Update windows.md (#1740)
jellyfin/jellyfin.org: docs/general/installation/windows.md

1. Go to [Add or remove programs](https://support.microsoft.com/en-us/windows/uninstall-or-remove-apps-and-programs-in-windows-4b55f974-2cc6-2d2b-d092-5905080eaf98) in Windows settings.
2. Search for Jellyfin.
3. Click Uninstall.

## Installing Jellyfin as a Windows Service (Advanced)

Running as a **Network Service** is preferred over **Local System** for the principle of least privilege. Follow these steps to ensure proper access without over-provisioning permissions.

### 1. Account & Ownership Prep
* **Installer:** Select "Install as a Service" & **uncheck** "Start Service after Install".
* **Service Account:** Set to `Network Service`.

> * **Folder Ownership:** Ensure your **Admin user** retains ownership of your media directories.

Reviewer: There is missing whitespace around the headings, if you run `npm run markdown:lint` these headings show up as linting errors. To fix, you just need to add space around the headings.

```suggestion
### 1. Account & Ownership Prep

* **Installer:** Select "Install as a Service" & **uncheck** "Start Service after Install".
* **Service Account:** Set to `Network Service`.
* **Folder Ownership:** Ensure your **Admin user** retains ownership of your media directories.
```

### 2. Configure Jellyfin Data Permissions
The service account needs access to Jellyfin's configuration files.

> 1. Navigate to `C:\ProgramData\`.

Reviewer: Missing whitespace, and suggest to remove numbering on heading. This heading might be more suited as "Configure Service Account Permissions"

```suggestion
### Configure Service Account Permissions

The service account needs access to Jellyfin's configuration files.

1. Navigate to `C:\ProgramData\`.
```

2. Right-click the **Jellyfin** folder > **Properties** > **Security** > **Advanced**.
3. **Change Owner:** Set to `NETWORK SERVICE`.
4. **Add Principal:** `NETWORK SERVICE`.
5. **Permissions:** Grant **Full Control** or **Modify**.
6. **Inheritance:** Check **"Replace all child object permission entries..."**
7. **Apply** and exit.

### 3. Configure Media Library Permissions (Least Privilege)
Grant the service account read-only access to your media to prevent "Path not found" errors.
1. Right-click your media folder > **Properties** > **Security** > **Advanced**.
2. **Add Principal:** `NETWORK SERVICE`.
3. **Permissions:**
   * **Basic:** Read & execute, List folder contents, Read.
4. **Inheritance:** Check **"Replace all child object permission entries..."**
5. **Apply** and exit.

### 4. Verify & Initialize
* **Service Check:** Open `services.msc`, right-click **Jellyfin Server**, and select **Start**. Verify the "Log On As" column shows `Network Service`.

> * **Access Web UI:** Navigate to [http://localhost:8096](http://localhost:8096) to complete the wizard.

Reviewer: I suggest you use numbering instead of bullet points to match the other sections, and remove the number from the header. Additionally, you might want to say how to open services.msc instead of just saying "open it."

```suggestion
### Verify and Initialize

1. Press **Windows**+**R** and type "services.msc". Hit **Enter**.
2. Find the **Jellyfin Server** service and verify the "Log On As" column shows **Network Service**.
3. Right-click **Jellyfin Server** service and select **Start**.
4. Navigate to http://localhost:8096 to complete the wizard.
```

> [!TIP]
> If libraries appear empty, ensure the `NETWORK SERVICE` has at least "List Folder" permissions on any parent directories leading to your media folder.

> [!NOTE]

> > To allow remote access to Jellyfin server, add a firewall rule that accepts `inbound` traffic for port `8096`.

Reviewer: These don't render correctly.

<img width="1201" height="372" alt="Image" src="https://github.com/user-attachments/assets/a17382aa-6156-426b-9d6a-6d318bff35de" />
